src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="../src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__reviews">
```

Comment:
```suggestion
      <p class="card__code">
        Product code: 195434
      </p>

      <div class="card__reviews">
```
there is nested content in siblings - add a blank line ([see](https://github.com/mate-academy/layout_product-cards/pull/3214#discussion_r1233016656))

fix all similar cases